src/Nethermind/Nethermind.Consensus.AuRa/InitializationSteps/InitializeBlockchainAuRa.cs

```diff
@@ -128,6 +128,8 @@ private IAuRaValidator CreateAuRaValidator(IBlockProcessor processor, IReadOnlyT
                 _api.LogManager,
                 chainSpecAuRa.TwoThirdsMajorityTransition);
 
+            NonceManager nonceManager = new(_api.StateProvider);
```

Comment:
Now in Aura you have multiple NonceManagers, you probably wanted to have just one?